components/layout/ErrorPageLayout.tsx

```diff
@@ -28,13 +29,15 @@ export const ErrorPageLayout = ({
     <PageContainer id={pageContainerId}>
       <div className="flex flex-col items-center text-center">
         <Image src={imgError} alt={imgAlt} width={400} height={417} loading="eager" className="bt-5 mb-10" />
-        <h1 className="mb-4 text-7xl font-extrabold tracking-tight text-violet-600 lg:text-9xl">Error</h1>
```

Comment:
Same issue for error pages bigger fonts.

Reply:
Fixed with: https://github.com/krsiakdaniel/portfolio-website-krsiak-cz/pull/172/files